addons/l10n_se_ocr/data/mail_template_data.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
How is this different from the standard template?

Reply:
Well you could have looked it up, but here it is.

```
<div style="margin: 0px; padding: 0px;">
    <p style="margin: 0px; padding: 0px; font-size: 13px;">
        Dear
        % if object.partner_id.parent_id:
            ${object.partner_id.name} (${object.partner_id.parent_id.name}),
        % else:
            ${object.partner_id.name},
        % endif
        <br /><br />
        Here is your
        % if object.name:
            invoice <strong>${object.name}</strong>
        % else:
            invoice
        %endif
        % if object.invoice_origin:
            (with reference: ${object.invoice_origin})
        % endif
        amounting in <strong>${format_amount(object.amount_total, object.currency_id)}</strong>
        from ${object.company_id.name}.
        % if object.invoice_payment_state == 'paid':
            This invoice is already paid.
        % else:
            Please remit payment at your earliest convenience.
        % endif
        <br /><br />
        Do not hesitate to contact us if you have any questions.
    </p>
</div>

```
New template:
```
<div style="margin: 0px; padding: 0px;">
    <p style="margin: 0px; padding: 0px; font-size: 13px;">
        Dear ${object.partner_id.name}
        % if object.partner_id.parent_id:
            (${object.partner_id.parent_id.name})
        % endif
        <br /><br />
        Here is, in attachment, your 
        % if object.name:
            invoice <strong>${object.name}</strong>
        % else:
            invoice
        % endif
        % if object.invoice_origin:
            (with reference: ${object.invoice_origin} )
        % endif
        amounting in <strong>${format_amount(object.amount_total, object.currency_id)}</strong>
        from ${object.company_id.name}.
        % if object.invoice_payment_state=='paid':
            This invoice is already paid.
        % else:
            Please remit payment at your earliest convenience.
            % if object.invoice_reference_type=='structured' and object.ref:
                Please use the following communication for your payment: <strong>${object.ref}</strong>.
            % endif
        % endif
        <br /><br />
        Do not hesitate to contact us if you have any questions.
    </p>
</div>
```

So the difference is actually pointing out that the OCR number should be used. But I will correct the template since I notice two things I forgot to change.

Reply:
After changes the difference will be a new section after "Please remit payment at your earliest convenience." telling the user to use the ocr number as payment reference.

Reply:
The difference after next update is that the following is added in the e-mail:
                            % if object.journal_id.invoice_reference_model in ['se_ocr2', 'se_ocr3', 'se_ocr4'] and object.invoice_payment_ref:
                                <br /><br />
                                Please use the following communication for your payment: <strong>${object.invoice_payment_ref}</strong>.
                            % endif

Reply:
These email templates are not the easiest to inherit and I wonder why we can not just add it in generic.  The same was done in l10n_be_invoice_bba and it might have been for historical reasons.  I need to investigate.

Reply:
They should be added generic if possible. It's good to have payment information in the e-mail that is sent to the customer, even if you are not using OCR or BBA.

Reply:
You are right.  I am working on it: https://github.com/odoo-dev/odoo/commit/919b5741ffdd21237513ed985cb99b8276f04a6d

Reply:
Ok. Removing template from commit.